connector/connector.py

```diff
@@ -49,42 +46,18 @@ def _get_openerp_module_name(module_path):
 
 
 def install_in_connector():
-    """ Installs an OpenERP module in the ``Connector`` framework.
+    log_deprecate("This call to 'install_in_connector()' has no effect and is "
+                  "not required.")
 
-    It has to be called once per OpenERP module to plug.
 
-    Under the cover, it creates a ``openerp.models.AbstractModel`` whose
-    name is the name of the module with a ``.intalled`` suffix:
-    ``{name_of_the_openerp_module_to_install}.installed``.
+def is_module_installed(env, module_name):
+    if env.registry.get('connector.backend') is None:
```

Comment:
So, the generic use case does not work otherwise we would not need to check the existence of the model. And the second test is made redundant by the first test for this specific use case.